### ISSUE TITLE
Incompatibility with 2021-09

### ISSUE THREAD

**Issue by guw** [MoreUnit/MoreUnit-Eclipse#126]
Looks like latest JDT has some internal changes which broke MoreUnit.

```
java.lang.NoSuchMethodError: 'void org.eclipse.jdt.internal.junit.wizards.MethodStubsSelectionButtonGroup.<init>(int, java.lang.String[], int)'
	at org.moreunit.wizards.MoreUnitWizardPageOne.<init>(MoreUnitWizardPageOne.java:165)
	at org.moreunit.wizards.NewTestCaseWizard.addPages(NewTestCaseWizard.java:51)
```

**Comment by apupier** [comment]
Can you check which version of MoreUnit you are using please? This should have been fixed with MoreUnit 3.3+

Based on line numbers, it sound slike you are using an older version
curent is pointing to a blank 165 line https://github.com/MoreUnit/MoreUnit-Eclipse/blob/105df27e70e210668be03585c76515b8edc17abc/org.moreunit.plugin/src/org/moreunit/wizards/MoreUnitWizardPageOne.java#L165
3.2.1 seems to corresponds https://github.com/MoreUnit/MoreUnit-Eclipse/blob/51b6e7fd4d5bdb6b5a8b33cf08b4e23b44fa169c/org.moreunit.plugin/src/org/moreunit/wizards/MoreUnitWizardPageOne.java#L165

**Comment by apupier** [comment]
there is now CI job with latest Eclipse 2021-09 and 2021-12 https://github.com/MoreUnit/MoreUnit-Eclipse/runs/4691492277?check_suite_focus=true which are green.
I'm closing as I think that it is because MoreUnit 3.2 is used.
